prow/plugins/trigger/generic-comment.go

```diff
@@ -30,6 +30,7 @@ import (
 var okToTestRe = regexp.MustCompile(`(?m)^/ok-to-test\s*$`)
 var testAllRe = regexp.MustCompile(`(?m)^/test all,?($|\s.*)`)
 var retestRe = regexp.MustCompile(`(?m)^/retest\s*$`)
+var genericTestRe = regexp.MustCompile(`(?m)^/test (?:.*? )?.+(?: .*?)?$`)
```

Comment:
Currently we can't assume that a triggering commands start with `/test`. That is just the default. Any regexp can be specified for a presubmit's `trigger` field.
If we want to start requiring that we probably could, but it would need to be documented and enforced when parsing config. It would also be a breaking change.  e.g. https://github.com/kubernetes/test-infra/issues/10945#issuecomment-458834788

Reply:
Great point.